### DISCUSSION TITLE
Allow approval requirements to be conditional

### DISCUSSION THREAD

**Discussion by LarsKemmann** [CareTogether/CareTogetherCMS#95] [Ideas]
This will allow, for example, pool safety checklists to only be required if a volunteer host family has a pool.

Requires adding policy-driven data attributes to a volunteer family, first, before the conditional logic can be implemented.

**Comment by simply-emily** [comment]
Yes, please! This should be for things such as firearms, pool/waterfront

**Comment by LarsKemmann** [comment]
Note to self: support having an "indeterminate" state initially.

Gun, water (pool/pond/etc.), animals (to control rabies vaccine requirements)

**Comment by AGNMM18** [comment]
Can we add out of state CPS checks?

**Comment by AGNMM18** [comment]
Note from NMM Director:

My preference is that all items be listed at the beginning so that we know what still needs to be addressed.  Does that make sense?

My goal for the approval screen is that we can see all that is left to collect before the approval process is complete.  If we add it later, a reviewer will not know that it is still left to address.

**Comment by LarsKemmann** [comment]
Here's how I think we can address this. Design concept discussed at tonight's call. Fields start out blank, a selection is required for each field before any _family role_ can go to Approved status, and requirements can be made conditional based on a corresponding field:
![image](https://user-images.githubusercontent.com/5834429/143962837-a0f40fc8-8b04-4996-9138-f9a077c0b99f.png)
